bridge/client/main.ts

```diff
@@ -1,4 +1,4 @@
-import 'hammerjs';
```

Comment:
Is this related to the Angular Upgrade, or did we actually not need this at all?

Reply:
https://indepth.dev/posts/1075/exciting-times-ahead-be-ready-for-angular-9#hammerjs-is-now-optional
It was removed by the update process. Also hammerjs is not used in bridge.